### ISSUE TITLE
[BUG] - XML error

### ISSUE THREAD

**Issue by asmith3006** [AndyDLP/PSDokuWiki#37]
**Describe the bug**
When updating content the content is saved but the response is:
```
[error] Exception: Cannot convert value "    <?xml version="1.0"?>
<methodResponse>
  <params>
    <param>
      <value>
      <boolean>1</boolean>
      </value>
    </param>
  </params>
</methodResponse>
" to type "System.Xml.XmlDocument". Error: "Unexpected XML declaration. The XML declaration must be the first node in the document, and no whitespace characters are allowed to appear before it. Line 1, position 7." 
```

I think this is likely due to spaces being returned in the XML from Dokuwiki, 

A possible solution would be:
```powershell
$response = @"
   <?xml version="1.0"?>
<methodResponse>
  <params>
    <param>
      <value>
      <boolean>1</boolean>
      </value>
    </param>
  </params>
</methodResponse>
"@

[System.Xml.XmlDocument]$response # Error
[System.Xml.XmlDocument]($response.Trim()) #Success
```

**Comment by asmith3006** [comment]
Resolved with https://github.com/AndyDLP/PSDokuWiki/pull/38

**Comment by AndyDLP** [comment]
fixed by #38
